javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java

```diff
@@ -0,0 +1,982 @@
+            case "1":
+                metricOption= statTypes.get(0);
+                break;
+            case "2":
+                metricOption= statTypes.get(1);
+                break;
+            case "3":
+                metricOption= statTypes.get(2);
+                break;
+            case "4":
+                metricOption= statTypes.get(3);
+                break;
+            case "5":
+                metricOption= statTypes.get(4);
+                break;
+            default:
+                System.out.println("You did not select a valid option.");
+                System.exit(1);
+        }
+        System.out.println("You selected "+metricOption);
+        getAndDisplayMetricStatistics(cw, selectedNamespace, selectedMetrics, metricOption, myDate, myDimension);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("4. Get CloudWatch estimated billing for the last week.");
+        getMetricStatistics(cw, costDateWeek);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("5. Create a new CloudWatch dashboard with metrics.");
+        createDashboardWithMetrics(cw, dashboardName, dashboardJson);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("6. List dashboards using a paginator.");
+        listDashboards(cw);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("7. Create a new custom metric by adding data to it.");
+        createNewCustomMetric(cw, dataPoint);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("8. Add an additional metric to the dashboard.");
+        addMetricToDashboard(cw, dashboardAdd, dashboardName);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("9. Create an alarm for the custom metric.");
+        String alarmName = createAlarm(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("10. Describe 10 current alarms.");
+        describeAlarms(cw);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("11. Get current data for new custom metric.");
+        getCustomMetricData(cw,settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("12. Push data into the custom metric to trigger the alarm.");
+        addMetricDataForAlarm(cw, settings) ;
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("13. Check the alarm state using the action DescribeAlarmsForMetric.");
+        checkForMetricAlarm(cw, settings);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("14. Get alarm history for the new alarm.");

```

Comment:
The alarm history output is all errors because it is using the dummy ids. Does it work with real ids so we can be sure the alarm is working?
Example: Failed to execute action arn:aws:sns:us-east-1:1234567890:Default_CloudWatch_Alarms_Topic